mousestyles/behavior/construct_trees.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function, absolute_import, division
+
+from .behavior_tree import BehaviorTree
+from mousestyles.behavior import metrics
+
+
+def process_raw_intervals(consumption, intervals, active_time,
+                          total_time, epsilon):
+    """
+    Takes `total_consumption` and the data in `intervals` and
+    decomposes it into a tree of behavioral summary statistics.
+
+    Parameters
+    ----------
+    consumption: float
+        total consumption (food, water, or movement) in the time period
+    intervals: Intervals
+        Intervals object representing intervals of behavior
+    epsilon: float
+        tolerance for merging events into bouts
+
+    Returns
+    -------
+    behavior : BehaviorTree
+        dictionary-like object representing breakdown of
+        consumption into various nodes
+
+    Examples
+    --------
+    >>> from mousestyles import data, intervals
+    >>> ints = intervals.Intervals(data.load_intervals('AS'))
+    >>> results = process_raw_intervals(1000, ints, 1)
+    """
+    tree = BehaviorTree(['Consumption',
+                         ['AS Prob',
+                          ['Intensity',
+                           ['Bout Rate',
+                            ['Bout Size',
+                             ['Bout Duration',
+                              ['Bout Intensity',
+                               ['Bout Event Rate',
+                                'Event Size']]]]]]]])
+    bout_intervals = metrics.create_collapsed_intervals(intervals, epsilon)
+    as_probability = active_time / total_time
+    tree['Consumption'] = consumption
```

Comment:
The top level isn't actually a consumption amount, but a consumption rate. See the photo attached from our meeting. 
So it should be 
```python
tree['Consumption Rate'] = consumption / total_time
tree['Intensity'] = consumption / active_time
```
Others that are incorrectly defined at the moment:
```python
tree['Bout Rate'] = bout_intervals.num() / active_time
tree['Bout Duration'] = bout_intervals.measure() / bout_intervals.num() #maybe save bout_intervals.num() in a temp variable
tree['Bout Intensity'] = tree['Bout Size'] / tree['Bout Duration']
```

![image](https://cloud.githubusercontent.com/assets/1553821/15100274/10a7105a-1522-11e6-9eb1-6d45f1973f0c.png)